js_modules/dagit/packages/core/src/runs/RunUtils.tsx

```diff
@@ -65,24 +64,17 @@ export type LaunchBehavior = 'open' | 'open-in-new-tab' | 'toast';
 
 export function handleLaunchResult(
   pipelineName: string,
-  result: void | {data?: LaunchPipelineExecution | LaunchPipelineReexecution | null},
+  result: void | null | LaunchPipelineExecution_launchPipelineExecution,
```

Comment:
Might there be any problems with this type with regard to `LaunchPipelineReexecution`? E.g. is there a parent or root run ID that needs to be taken into account here? It looks like TS passes, so maybe it's fine?

Reply:
Yeah it's fine since we're not accessing any of the fields specific to the underlying types here